flyway-community-db-support/src/main/java/org/flywaydb/community/database/clickhouse/ClickHouseTable.java

```diff
@@ -0,0 +1,37 @@
+package org.flywaydb.community.database.clickhouse;
+
+import lombok.CustomLog;
+import org.flywaydb.core.internal.database.base.Table;
+import org.flywaydb.core.internal.jdbc.JdbcTemplate;
+
+import java.sql.SQLException;
+
+@CustomLog
+public class ClickHouseTable extends Table<ClickHouseDatabase, ClickHouseSchema>
+{
+    /**
+     * @param jdbcTemplate The JDBC template for communicating with the DB.
+     * @param database The database-specific support.
+     * @param schema The schema this table lives in.
+     * @param name The name of the table.
+     */
+    public ClickHouseTable(JdbcTemplate jdbcTemplate, ClickHouseDatabase database, ClickHouseSchema schema, String name) {
+        super(jdbcTemplate, database, schema, name);
+    }
+
+    @Override
+    protected void doDrop() throws SQLException {
+        jdbcTemplate.executeStatement("DROP TABLE " + database.quote(schema.getName(), name));
```

Comment:
Some cases require `ON CLUSTER <cluster_name>` suffix, from ClickHouseConfigurationExtension

Reply:
Thanks. Fixed that.